airtime_mvc/application/models/Preference.php

```diff
@@ -201,10 +201,11 @@ private static function getValue($key, $isUserValue = false, $forceDefault = fal
     public static function GetHeadTitle()
     {
         $title = self::getValue("station_name");
-        if (strlen($title) > 0)
-            $title .= " - ";
-
-        return $title.PRODUCT_NAME;
+        if ($title == null) {
```

Comment:
This branch already had merge conflicts when I created #871. I guess landing this might require a bit of effort since it seems to have quite some changes unrelated to the core feature being implemented in the branch.

Reply:
Yes, that's my fault for not having seperate pull requests. I did the PR before it and suggested it in discourse, but I can just remove it and merge the one hairmare added since it's much cleaner.